app/(main)/graduation-progress/[studentCode]/[department]/page.tsx

```
@@ -221,7 +221,7 @@
             isCompleted={area.earned_credits >= area.required_credits}
             courses={area.courses}
           />
-          {index < areaProgress.length - 1 && <div className={styles.gap12}></div>}
+          {index < areaProgress.length - 1 && <div className="gap-12"></div>}
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)